### PR TITLE
New Features: Added Footer Section and Our Team Section

### PR DESCRIPTION
**Pull request by rudrathegreat**
As @ConvexPolygon was having some trouble trying to produce results in the time he would finish it in, I decided to help out. I designed a simple Our Team Section, and I also designed a footer. All features and enhancements are below -

- Created Footer
- Made Footer Responsive based on viewport width
- Made Footer responsive based on viewport height
- Added Additional Mobile Support for the Footer
- Added Additional Mobile Support for Latest News Section
- Developed 'Meet Our Team' section with join today button
- Made Meet Our Team section responsive based off viewport width
- Made Meet Our Team section responsive based off viewport height
- Added Additional Mobile Support for Meet Our Team Section
- Made Navbar smaller
- Reduced Opening animation by 1 sec

Let me know your thoughts. Again, similar to the Latest News Section, there are only placeholders for both the footer and the Meet Our Team Section. I recommend that everyone adds their name, one-word description and contacts. Also add your logo/photo to replace the sc4 logo. As for the footer, there is a section called the 'Latest Updates' which briefly mentions the news as well. Also put relevant info there. Again, let me know your thoughts ASAP and put it in the develop branch until NO placholders remain and the site is filled with working links.

### CODE REVIEW

**Comment by rudrathegreat** [comment]
@micahlt @githubfan pls check this out!

**Comment by micahlt** [comment]
@rudrathegreat I'll review this ASAP.

**Comment by locness3** [comment]
The user links in the "meet the team" section are unclickable on an actual phone
IMG_1554

**Comment by rudrathegreat** [comment]
@locness3 the links don't work because I have not added any links yet, that is why I told the reviewers to take note to merge it into the develop branch, because I want everyone to add their own social media links, their name, their tagline, etc. And I believe that the website should be based off viewport width as it is independent to screen size (1080p, 4k, etc.), pixel density, etc. making it more responsive on a variety of devices.

**Comment by locness3** [comment]
I know the user links are placeholders, but look at their size, proportionally to my finger (I sent a picture, have you seen it?) @rudrathegreat

**Comment by rudrathegreat** [comment]
I've seen that and I am making changes rn btw i was at school!

**Comment by rudrathegreat** [comment]
yeah they look really tiny, i think instead we have a sort of onclick effect in which there is layer which comes on top of the card, showing the contact links, that should do the trick. Requires javascript though, working on it rn!

**Comment by locness3** [comment]
I think the best is to stack them on mobile.

**Comment by rudrathegreat** [comment]
Ok, made some changes, hopefully this can be merged to the main branch

File: index.html [hunk 1]
```diff
@@ -42,145 +52,301 @@ <h1>Meet Our Team</h1>
...
-              <p>Developer</p>
-              <ul>
+            <td onclick='showLinks("member2", "links2");'>
+              <div id="member2" class="showMember">
+                <img src="assets/square.png">
+              <h3>Micah</h3>
+              <p>Admin</p>
+              </div>
+              <div id="links2" class="hide">
+                <ul style="padding-top:20%">
```
Comment (locness3) [inline index.html:82]: Please avoid inline styles

File: index.html [hunk 2]
```diff
@@ -42,145 +52,301 @@ <h1>Meet Our Team</h1>
...
-
     </div>
     </div>
   <div class="news" id="news">
     <div class="news-container">
       <h1>See the latest news with our apps!</h1>
       <hr>
-      <h2>Lorem Ipsum Title</h2>
-      <p>Lorem Ipsum Description which contains a one-line summary of the news.</p>
+      <h2>v0.2.4 Released!</h2>
```
Comment (locness3) [inline index.html:348]: 0.2.4 of what?

File: index.html [hunk 3]
```diff
@@ -24,6 +26,14 @@
       </ul>
     </nav>
   </div>
+  <script>
```
Comment (locness3) [inline index.html:29]: jquery bad

**Comment by locness3** [comment]
"documentation"? 🤔

File: index.html (hunk 1 above)
Comment (rudrathegreat) [reply index.html:82]: k understood making changes now

File: index.html (hunk 2 above)
Comment (rudrathegreat) [reply index.html:348]: v0.2.4 of the scratch client 4 app

File: index.html (hunk 3 above)
Comment (rudrathegreat) [reply index.html:29]: ouch dude :( wdym by bad?

**Comment by rudrathegreat** [comment]
oh and btw @neevjew is working on the click effect I described above (that's going to fix the tiny buttons problem for mobile)

File: index.html (hunk 2 above)
Comment (locness3) [reply index.html:348]: Which one, desktop or mobile

File: index.html (hunk 3 above)
Comment (locness3) [reply index.html:29]: Erm well jquery is ok but it's a bit old and a bit too much. Raw JavaScript should be able to do the job, and if you want to type less you can assign characters to raw js functions, for example $ and $a to `querySelector` and `querySelectorAll` this way :
```
$ = document.querySelector.bind(document);
$a = document.querySelectorAll.bind(document);
```
I haven't looked into the code, though, so i don't know what uses jquery.

**Comment by locness3** [comment]
Why is there a "documentation" label?

**Comment by micahlt** [comment]
Is this ready for merge?

**Comment by locness3** [comment]
I requested changes.

**Comment by rudrathegreat** [comment]
I left the org. Sorry couldn't get time to work on anything

**Comment by locness3** [comment]
Scratch Client 4 really isnt anything anymore. Thanks for trying to help.

**Pull request closed without merging**